README.md

```diff
@@ -1,2 +1,64 @@
-# British-Classic-Meals
-This App is about British Food where you find a list of meals that you can like and comment.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# British Classic Meals
+The British Classic Meals is a seclection of the most favorite and famous british recipes.
+Things you can do in this website:
+- check the list of meals
+- like the recipe
+- view the complete recipe including list of ingredients
+- view comments of other users
+- leave your comment about the meal
+
+Check it out! What is your favorite from the list? Or which one would you choose to eat now if you could?
+
+## Built With
+
+- CSS
+- HTML
+- JavaScript
+- Webpack
+
+## Live demo
+You can access the live demo through this [link](https://tiago-lelinski-marin.github.io/British-Classic-Meals/)
+
+##  Video Link
+You can access the video link through this [link](https://drive.google.com/file/d/12CF-P4dRWUbA8ktLYeDWfYrq3myBnbdw/view?usp=sharing)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps:
+
+- Clone this repo to your computer using git.
+Or
+- Download the compressed [zip folder](https://github.com/Tiago-Lelinski-Marin/British-Classic-Meals/archive/refs/heads/dev.zip) and use a decompressor to unzip it into your computer. Then open the folder and all the project files will be there.
+
+- then instal npm in with "npm install" and then **use "npm run start"** to open the application on your browser
+
+## Authors
+
+👤 **Tiago Lelinski Marin**
+
+- GitHub: [@githubhandle](https://github.com/Tiago-Lelinski-Marin)
+- Twitter: [@twitterhandle](https://twitter.com/LelinskiMarin)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/tiago-lelinski-marin/)
+
+👤 **Basit Ali**
+
+- GitHub: [@githubhandle](https://github.com/basitali111)
+- Twitter: [@twitterhandle](https://twitter.com/BasitAl35031734)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/basit-ali-3961141b3/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [OPTIONAL] When I click the link to the `MIT.md` file that you added, I get this 404 `error`. Please update your Readme, it will be nice to update this section(`License`) with the type of License and a link to the license file, as this will make your readme more informative and professional. Check out this [template](https://github.com/microverseinc/readme-template) for more clarifications